web/src/connectors/edit-job-connector/edit-job-connector.tsx

```diff
@@ -116,6 +121,13 @@ const EditJobConnector: FC<EditJobConnectorProps> = ({
             };
         }
 
+        if (validationType === 'validation only') {
```

Comment:
I suggest to use "ValidationType" enum instead.

Reply:
fixed